app/services/create_api_key.rb

```diff
@@ -54,6 +54,7 @@ def create_in_api_gateway(api_key)
       name: api_key.api_key_id,
       description: api_key.description,
       enabled: api_key.enabled,
+      value: api_key.secret,
```

Comment:
All this time we've been displaying the wrong thing? 😭

Reply:
I did some live validations in dev and the secret wasn't working - can't believe I missed this lol